Embedded Link -> Link (#1031)
codatio/codat-docs: docs/auth-flow/authorize-hosted-link.md

---
title: "Authorize with Hosted Link"

> description: "See how to integrate Hosted Link into your authorization flow"

Reviewer: Is this page purposefully excluded from the side nav?

unlisted: true
displayed_sidebar: docs
---

<head>
  <meta property="og:image" content="/img/link/link-banner.png"/>
</head>

:::tip Codat recommends Link SDK

Instead of Hosted Link, use the [Link SDK](/auth-flow/authorize-embedded-link) to fully embed our flexible, white-labeled auth flow in your application. 

You will benefit from our extensive experience combined with authorization best practices, providing your users with a native feel of the auth flow and achieving an **89%** conversion rate.

::: 

## Overview

> If your business scenario and circumstances prevent you from using our Link SDK, you can choose Hosted Link as your out-of-the-box authorization solution built, provided, and hosted by Codat. To connect your customers' financial software, you can:

Reviewer: ```suggestion
If you're not able to use our Link SDK, you can choose Hosted Link as your out-of-the-box authorization solution built, provided, and hosted by Codat. 

To connect your customers' financial software with Hosted Link, you can:
```

> - Integrate the Hosted Link auth flow into the user interface of your existing app, or

Reviewer: I guess here you're trying to say they can either:
- Manually share a URL
- Redirect the user programmatically to the hosted link
?

Maybe the word 'integrate' is a little vague in this case
